### DISCUSSION TITLE
No longer possible to edit shortcut

### DISCUSSION THREAD

**Discussion by minff** [AppGround-io/bluetooth-keyboard-and-mouse-support#328] [Q&A]
Before there is an edit button to edit shortcut buttons. Since version 6.x the button disappeared. Is this by design ? How do I edit shortcut now ?

One related but outdate thread: https://github.com/AppGround-io/bluetooth-keyboard-and-mouse-support/discussions/141

**Comment by ldca58** [comment]
Same problem here. I cannot find a way to edit the buttons any more. Is it a bug, or am I missing something?

**Comment by AppGround-io** [comment]
The shortcuts feature, an earlier version of custom layouts, will be removed soon. To ease the transition, only editing has been disabled for now. Please use the custom layouts feature for more customization options.

**Reply by ldca58** [reply]
Thank you for the information. To make the transition easier, would it be possible to have an option to copy the shortcuts to a custom layout?
